index.rst

```diff
@@ -48,11 +48,256 @@
+This will be reflected in the constant (:math:`C_0`) term in the elevation LUT.
+
+The hexapod position control has a basic operation mode called the *LUT mode*.
+By default, the LUT mode is off, #1 above is set to zero.
+This is useful for engineering testing. In this mode, the hexapod should simply move to the user-commanded position (depending on whether it is a move (``cmd_move``) or offset (``cmd_offset``) command, it will be in reference to either (0,0,0,0,0,0) or the current position of the hexapod).
+When the LUT mode is on, #1 above is determined by the LUTs.
+This is useful for science operations. A science user will not have to deal with or even think about how the LUT works.
+The hexapod CSC will take care of that in the background.
+For example, when a science user wants to take an extra-focal image at 1mm, the command is simply ``cmd_move.set_start(z=1000)``.
+The CSC will be contantly adjusting the LUT compensation taking into account changes in the elevation angle and temperature etc.
+
+Note that there are a few *special* spacial points in terms of operating a hexapod.
+
+- The (0,0,0) as determined by the hexapod encoders - All the positions reported by the hexapod control system and CSC are in reference to a coordinate system (CS) with this point as origin.
+- The pivot point, as defined by the user using command ``cmd_pivot`` - the input x, y, and z to this command is defined in the CS above. For AOS operations, we will mostly use L1S1 first vertex and M2 vertex as pivot points for the Camera and M2 hexapods, respectively;
+- The collimated position, which is defined by the LUT. It puts L1S1 first vertex and M2 vertex roughly at the origins of the CCS or M2CS, defined in reference to M1M3.
+
+Below we use an example to demonstrate how these commands are supposed to be used. For simplicity, we assume there is only one degree of freedom which is z displacement; the unit is arbituary; and there is only one LUT which only depends on elevation. When the hexapod was last disabled, it was at position z=5.
+
+#. Once we enter enabled state, the hexapod stays at z=5, LUT mode = off;
+#. The user turns LUT on using ``cmd_enableLut``, the hexapod stays at z=5; but all subsequent move commands will take into account the LUT;
+#. Then the user sends command ``cmd_move.set_start(z=-1)``. The hexapod moves to z=-1+3.1=2.1. The 3.1 is from the LUT and calculated using the elevation angle;
+#. The user sends command ``cmd_move.set_start(z=1)``, elevation is unchanged, the hexapod goes to z=1+3.1 = 4.1.
+#. Elevation changes result in LUT compensation to change to 3.2, the hexapod moves to z=4.2.
+#. The user issues command ``cmd_offset.set_start(z=-1)``, the hexapod moves to z=4.2-1 = 3.2.
+#. The user turns off compensation mode using ``cmd_enableLut``, the hexapod stays at z=3.2.
+
```

Comment:
This example is great.  However, this bring the  potential issue of taking into account the laser tracker original  offsets. We could use C0 that you mentioned earlier, but would that require to make a change to the LUT without a change review?